project/assets/test/ui/level-select/career-worlds-simple.json

```diff
@@ -17,6 +17,9 @@
         "observer_35 30%",
         "(quirky) observer_30"
       ],
+      "demographics": [
+        "squirrels 100%"
```

Comment:
is it 'squirrel' or 'squirrels'? This conflicts with the data in `career-worlds.json`. Did you run the tests?

Reply:
Changed to `squirrel` -- the unit tests caught this problem and were failing.